core/sys/windows/user32.odin

```diff
@@ -32,6 +32,8 @@ foreign user32 {
 	RegisterClassExW :: proc(^WNDCLASSEXW) -> ATOM ---
 	UnregisterClassW :: proc(lpClassName: LPCWSTR, hInstance: HINSTANCE) -> BOOL ---
 
+	RegisterHotKey :: proc(hnwd: HWND, id: int, fsModifiers: UINT, vk: UINT) -> BOOL ---
```

Comment:
This should be `c.int` and add `import "core:c"` in the file. An Odin `int` is not the same as a c `int`.